gpt_index/vector_stores/utils.py

```diff
@@ -0,0 +1,12 @@
+import itertools
+from typing import Iterable, List
+
+
+def chunk_iterable(iterable: Iterable, batch_size: int) -> List[tuple]:
+    """A helper function to break an iterable into chunks of size batch_size."""
+    it = iter(iterable)
+    chunks = []
+    chunk = tuple(itertools.islice(it, batch_size))
```

Comment:
adding this line make it skip the first chunk (i just tested it locally) 

might be good to add a unit test